TestTools/StreamChatTestMockServer/MockServer/StreamMockServer.swift

```diff
@@ -16,6 +16,7 @@ public final class StreamMockServer {
     public private(set) var server: HttpServer = HttpServer()
     private weak var globalSession: WebSocketSession?
     private var channelConfigs = ChannelConfigs()
+    public var threadList: [[String: Any]] = []
```

Comment:
Does this mean we can only have one thread list? 🤔 What is the thread list dictionary used for? Couldn't each message in the `messageList` have a key-value of `replies: [[String: Any]]` ?

Reply:
Tbh, this `threadList` var was introduced only for the pagination scenario and it's used only when you'd like to generate loads of thread replies prior to the test. By default all replies live inside `messageList`, this just does not work with pagination, because it loads them straight away.

I could add a comment to this var, but not sure it makes sense to make `messageList` more complex just because of one scenario with pagination in the thread. WDYT?

Reply:
Makes sense then 👍